google/cloud/pubsub_v1/publisher/flow_controller.py

```diff
@@ -25,6 +25,9 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+MessageType = Type[types.PubsubMessage]  # type: ignore  # pytype: disable=module-attr
```

Comment:
I don't understand the `type: ignore` here, and I think this may be a misuse of `Type[]`.

Reply:
The ignore here is needed, because `PubsubMessage` is dynamically injected into `google/cloud/pubsub_v1/types.py` and both type checkers think it does not exist. The alias was created to not repeat the same ignore comment in every line where `message` is a method parameter.

Using a plain `types.PubsubMessage` in the alias results in `mypy` complaining that variable `MessageType` is not valid as a type. Using `Type[...]` works around that.

(alternatives welcome, especially if they are canonical)